smsish/sms/backends/twilio.py

```diff
@@ -0,0 +1,70 @@
+from django.conf import settings
+from django.core.mail.backends.base import BaseEmailBackend
+
+from twilio.rest import TwilioRestClient
+
+
+class SMSBackend(BaseEmailBackend):
```

Comment:
Use `username` and `password` here, and separate `SettingsSMSBackend` that works like this one (i.e. consults `django.conf.settings` for credentials)?